nightly.yml

```diff
@@ -10,10 +10,10 @@
       inputs:
           BuildDropPath: '$(Build.ArtifactStagingDirectory)\$(NugetArtifactsDir)\sbom'
 
-    - task: PublishBuildArtifacts@1
-      displayName: 'Publish SBOM'
-      inputs:
-        PathtoPublish: '$(Build.ArtifactStagingDirectory)\$(NugetArtifactsDir)\sbom'
+    # - task: 1ES.PublishPipelineArtifact@1
+    #   displayName: 'Publish SBOM'
+    #   inputs:
+    #     path: '$(Build.ArtifactStagingDirectory)\$(NugetArtifactsDir)\sbom'

```

Comment:
We generate the manifest in the previous 2 steps. If we're getting rid of this step, do we need to retain those 2 other steps?

Reply:
If we got rid of the other 2 steps, will there be any value in retaining this _nightly.yml_ file?

Reply:
The nightly also performs code signing